docs/docs/general/02-Getting Started/writing-your-first-test.md

```diff
@@ -19,7 +19,25 @@ This is going to create
   - `test/my-first-askui-test-suite.test.ts`: an example test with askui,
   - a folder `helper` which contains `jest.setup.ts` file: this file is setting up the test environment for all tests
   - a `jest.config.ts`: file for configuration of Jest,
-  
+
+### Configuration
+
+We need you to create some credentials through our [askui user portal](https://app.askui.com/) (usage is free!).
+Please see <a href="../askui%20User%20Portal/signup" target="_blank">our documentation on how to signup and create the credentials</a>.
+
+Then go to your `jest.config.ts` and add the configuration for your workspace Id and your access token to the _UiControlClient_.
```

Comment:
Minor thing but I would put a "," after "Then" and lowercase workspace id. 

On a more general note: If I go over the documentation, I can see that we all made a lot of spelling mistakes. Can we help us out using some tooling around that, e.g., 

- https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker
- https://www.npmjs.com/package/markdown-spellcheck
- https://marketplace.visualstudio.com/items?itemName=znck.grammarly
- npmjs.com/package/@grammarly/sdk

I am a big fan of Grammarly as it is from my perspective the market leader on improving the writing. Maybe, you can find a way to integrate it into our docs project so that it can help us out. And maybe even integrate it into our Githook and CI so that we have it in place as a check. What do you think?